CorsixTH/Lua/dialogs/queue_dialog.lua

```diff
@@ -176,16 +176,17 @@ function UIQueue:onMouseUp(button, x, y)
       return
     end
 
+    local first_patient_index = queue:size() - num_patients + 1
```

Comment:
Same calculation used in 169. Maybe move this up where num_patients is defined (or into queue)